### ISSUE TITLE
Configure application-specific constants that match the certification data

### ISSUE THREAD

**Issue by AlCalzone** [zwave-js/backlog#36]
This is required to specify which manufacturer ID etc. Z-Wave JS uses in responses. See https://github.com/zwave-js/node-zwave-js/pull/6876 for details. This info needs to be set by the integration:
```js
{
    // Nabu Casa
    manufacturerId: 0x0466,
    // Home Assistant, Z-Wave JS
    productType: 0x0001,
    productId: 0x0001,
    // optional, these are the defaults:
    hardwareVersion: 0,
    userIcon: 0x0500,
    installerIcon: 0x0500,
  }
```

**Comment by MartinHjelmare** [comment]
Is this only for users that are running the standard add-on? Previously we haven't set any driver options for other ways of running the server/driver from the integration, as far as I know.

**Comment by AlCalzone** [comment]
Correct. For certification, we need to be able to identify the combination HA+Core Addon+node-zwave-js to a Z-Wave network. The info above is how.

**Comment by MartinHjelmare** [comment]
This issue needs clarification both where this should be configured and on the values.

1. Can the Z-Wave add-on set these driver options, regardless of what controller is connected?
2. Why should we use NabuCasa for manufacturerId? Wouldn't the Open Home Foundation be more appropriate?

**Comment by AlCalzone** [comment]
1. Yes. This is about identifying the whole hub. The actual controller/adapter being used to communicate is irrelevant for this.
2. It would, but we haven't joined the Z-Wave Alliance with OHF.

**Comment by MartinHjelmare** [comment]
We'll wait with this until we know what manufacturerId to use.
